test/ssc_test/cmod_pvsamv1_test.cpp

```diff
@@ -743,9 +747,14 @@ TEST_F(CMPvsamv1PowerIntegration_cmod_pvsamv1, NonAnnual)
 {
     //set up a weather data array and unassign the solar resource file
 
-    auto weather_data = create_weatherdata_array(24);
+//    auto weather_data = create_weatherdata_array24(); // runs without error
+    auto weather_data = create_weatherdata_array(24); // throws SEH exception in release mode only from cmod_pvsamv1 on Windows
+   // var_data weather_data;
+   // create_weatherdata_array(weather_data, 24);
```

Comment:
Please remove or explain commented out code.

Reply:
These were added while trying to remove the occasional SEH test errors on Windows

Reply:
Reverted to develop and SEH windows errors occur with or without changes